external-crates/move/move-compiler/src/diagnostics/mod.rs

```diff
@@ -59,11 +59,11 @@ pub struct Diagnostics {
 pub enum WarningFilters {
```

Comment:
I think the main thing missing here is we have to domain separate the `WarningFilters`. As implemented today, the warning category/codes will overlap between external tools and the ones defined by the compiler.

Reply:
Indeed. A new version fixes this.